templates/cisco_ios_show_snmp_user.template

```diff
@@ -0,0 +1,17 @@
+Value Required USER_NAME (\w+)
+Value ENGINE_ID (\w+)
+Value STORAGE_TYPE (\w+)
+Value STORAGE_TYPE_ACTIVE (\w+)
```

Comment:
I think we should change the `STORAGE_TYPE_ACTIVE` to `ACCESS_LIST` to capture `d+` as that appears from the Cisco docs as to what it is referring to:

https://www.cisco.com/c/en/us/td/docs/ios-xml/ios/snmp/command/nm-snmp-cr-book/nm-snmp-cr-s1.html#wp8434359780

And then change L12 to:
`^storage-type:\s${STORAGE_TYPE}\s+active$$`

And add the following line right above:
`^storage-type:\s${STORAGE_TYPE}\s+active\s+access-list:\s+${ACCESS_LIST}$$`